api/openapi-spec/openapi.yaml

```diff
@@ -28,6 +28,23 @@ paths:
           description: Unauthorized, invalid JWT
         '403':
           $ref: '#/components/responses/Forbidden'
+  /v1/datafile:
+    get:
+      summary: Return the datafile for the given environment
+      operationId: getDatafile
+      description: Returns the json datafile for the given environment.
+      responses:
+        '200':
+          description: Valid response
+          content:
+            application/json:
+              schema:
+                description: Optimizely Datafile
+                type: object
```

Comment:
should it be object or string?

Reply:
We are returning datafile as json hence have written type as `object` here. If we intend to return datafile as `string` then we will have to map it against a key map to render it as json in response.